plugins/ssh-plugin/package.json

```diff
@@ -0,0 +1,51 @@
+{
+      "name": "@eclipse-che/theia-ssh-plugin",
+      "keywords": [
+        "theia-plugin"
+      ],
+      "version": "0.0.1",
+      "license": "EPL-2.0",
+      "contributors": [
+        {
+          "name": "Artem Zatsarynnyi",
+          "email": "azatsary@redhat.com"
+        },
+        {
+          "name": "Igor Vinokur",
+          "email": "ivinokur@redhat.com"
+        },
+        {
+          "name": "Vitalii Parfonov",
+          "email": "vparfono@redhat.com"
+        }
+      ],
+      "files": [
+        "src"
+      ],
+      "devDependencies": {
+        "@eclipse-che/api": "latest",
+        "@eclipse-che/plugin": "0.0.1",
+        "@theia/plugin": "next",
+        "@theia/plugin-packager": "latest",
+        "rimraf": "2.6.2",
+        "typescript-formatter": "7.2.2",
+        "typescript": "2.9.2",
```

Comment:
please remove all devDependencies that are already in the parent folder (yarn workspace)

Reply:
also probably axios is no longer used there ?

Reply:
fixed